ydb/docs/en/core/integrations/liquibase.md

```diff
@@ -0,0 +1,314 @@
+
+  ```bash
+  # $(which liquibase)
+  cd ./internal/lib/
+  
+  # you may need to sudo
+  # set an actual versions .jar files
+  curl -L -o ydb-jdbc-driver.jar https://repo1.maven.org/maven2/tech/ydb/jdbc/ydb-jdbc-driver-shaded/2.0.7/ydb-jdbc-driver-shaded-2.0.7.jar
+  curl -L -o liquibase-ydb-dialect.jar https://repo1.maven.org/maven2/tech/ydb/dialects/liquibase-ydb-dialect/1.0.0/liquibase-ydb-dialect-1.0.0.jar
+  ```
+  
+  For a more detailed description, see the [Manual library management](https://docs.liquibase.com/start/install/home.html ). 
+ 
+  Now the liquibase utility can be used in standard ways.
+
+{% endlist %}
+
+## Liquibase usage scenarios
+
+### Initializing liquibase to an empty {{ ydb-short-name }}
+
+The main command is 'liquibase update', which applies migrations if the current schema {{ ydb-short-name }} lags behind the user description.
+
+Let's apply this changeset to an empty database:
+
+```xml
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                      http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.8.xsd">
+
+    <changeSet id="episodes" author="kurdyukov-kir">
+        <comment>Table episodes.</comment>
+
+        <createTable tableName="episodes">
+            <column name="series_id" type="bigint">
+                <constraints primaryKey="true"/>
+            </column>
+            <column name="season_id" type="bigint">
+                <constraints primaryKey="true"/>
+            </column>
+            <column name="episode_id" type="bigint">
+                <constraints primaryKey="true"/>
+            </column>
+
+            <column name="title" type="text"/>
+            <column name="air_date" type="timestamp"/>
+        </createTable>
+        <rollback>
+            <dropTable tableName="episodes"/>
+        </rollback>
+    </changeSet>
+    <changeSet id="index_episodes_title" author="kurdyukov-kir">
+        <createIndex tableName="episodes" indexName="index_episodes_title" unique="false">
+            <column name="title"/>
+        </createIndex>
+    </changeSet>
+</databaseChangeLog>
+```
+
+After executing the `liquibase update` command, Liquibase will print the following log.
+
+```bash
+UPDATE SUMMARY
+Run:                          2
+Previously run:               0
+Filtered out:                 0
+-------------------------------
+Total change sets:            2
+
+Liquibase: Update has been successful. Rows affected: 2
+Liquibase command 'update' was executed successfully.
+```
+
+After applying migrations, the data schema now looks like this:
+
+![../_assets/liquibase-step-1.png](../_assets/liquibase-step-1.png)
```

Comment:
An `_assets` folder can be on any level in the documentation, and it's better to move these screenshots closer to the article to the `integrations` folder.